modules/cnf-intel-n3000-optimize-data-plane.adoc

```diff
@@ -0,0 +1,45 @@
+// CNF-1498 Validate and Document Intel SRO and SRIOV FEC Operator
+// Module included in the following assemblies:
+//
+// *cnf-optimize-data-performance-n3000.adoc
+
+[id="cnf-understanding-intel-hardware-accelertor-card-for-openshift-container-platform_{context}"]
+= Understanding Intel hardware accelerator cards for OpenShift Container Platform
+
+Hardware accelerator cards from Intel® accelerates 4G/LTE and 5G Virtualized Radio Access Networks (vRAN) workloads, increasing in turn the overall compute capacity of a commercial, off-the-shelf platform.
+
+The hardware accelerator cards detailed in this section are:
+
+* Intel® FPGA PAC N3000
+
+* Intel® vRAN Dedicated Accelerator ACC100
++
+[NOTE]
+====
+The Intel® FPGA PAC N3000 card needs flashed with the 4G/LTE or 5G bitstreams to support vRAN workloads.
+====
+
+.Intel® FPGA PAC N3000
+
+The Intel® FPGA PAC N3000 is a reference FPGA and uses 4G/LTE or 5G Forward Error Correction (FEC) as an example workload accelerating the 5G or 4G/LTE RAN layer 1 (L1) base station network function.
+
+The Intel® FPGA PAC N3000 is a full-duplex, 100 Gbps in-system, re-programmable acceleration card for multi-workload networking application acceleration. It has an optimal memory mixture designed for network functions, with an integrated network interface card (NIC) in a small form factor that enables high throughput, low latency, and low power per bit for a custom networking pipeline.
```

Comment:
How much is Intel paying us?  I think you can keep it to 100 Gpbs and that it is reprogrammable (the FP in FPGA), and then expect that the customer can find in-depth information about the hardware from the Intel website.